Multisig/Logic/Ethereum/GnosisSafe.swift

```diff
@@ -23,6 +23,20 @@ class GnosisSafe {
         ("0x34CfAC646f301356fAa8B21e94227e3583Fe3F5F", "1.1.1")
     ]
 
+    var fallbackHandlers:[(fallbackHandler: Address, label: String)] = [("0xd5D82B6aDDc9027B22dCA772Aa68D5d74cdBdF44", "DefaultFallbackHandler")]
```

Comment:
(App.shared.defaultFallbackHandler, "DefaultFallbackHandler")

Reply:
App.shared.defaultFallbackHandler is intended to be removed